src/routes/+page.svelte

```diff
@@ -1,2 +1,26 @@
-<h1>Welcome to SvelteKit</h1>
-<p>Visit <a href="https://kit.svelte.dev">kit.svelte.dev</a> to read the documentation</p>
+<script>
+	import Card from '$lib/Card.svelte';
+
+	export let data;
+	let isLoading = true;
+	let animes = data.animes;
+	//Use initialData for search functionnality
+	let initialData = data.animes;
+	isLoading = data.loading;
+</script>
+
+<div>
+	{#if isLoading === true}
```

Comment:
isLoading est un boolean, pas besoin de le comparer à true/false, c'est implicite
```suggestion
        {#if isLoading}
```